### PR TITLE
Lv2 pedals

### PR DESCRIPTION
**Pull request by steveb**
This PR adds and updates LV2 plugins which are useful for a moddevices guitar pedal

### CODE REVIEW

**Comment by guysoft** [comment]
Neat,
I am just noting that after review:
1. There is a TODO for moddevices to merge https://github.com/moddevices/caps-lv2/pull/17 and it might be worth to see if they are planning to merge before we merge, because it would effect where we point our cloning.
2. @steveb how come there are a bunch of commented out code that is changing there?
Fore example:`` # install_gxslowgear.sh`` to ``# install_gxboobtube.sh``

**Comment by steveb** [comment]
> Neat,
> I am just noting that after review:
> 
>     1. There is a TODO for moddevices to merge [moddevices/caps-lv2#17](https://github.com/moddevices/caps-lv2/pull/17) and it might be worth to see if they are planning to merge before we merge, because it would effect where we point our cloning.

I've asked in the PR. I think its OK to temporarily point at a different repo.

>     2. @steveb how come there are a bunch of commented out code that is changing there?
>        Fore example:` # install_gxslowgear.sh` to `# install_gxboobtube.sh`

GxSlowGear and GxVintageFuzzMaster are now provided by GxPlugins, so install_gxslowgear.sh etc were deleted files. However git decided the files were renamed. The commented `# install_gxboobtube.sh` is just a documentation convention I copied from existing files

**Comment by jujudusud** [comment]
Hi there, I think you should go here for updated caps-lv2 : https://github.com/BlokasLabs/caps-lv2

**Comment by steveb** [comment]
> Hi there, I think you should go here for updated caps-lv2 : https://github.com/BlokasLabs/caps-lv2

Done, thanks. BlokasLabs/caps-lv2 builds for me locally

File: scripts/recipes/install_guitarix.sh [hunk 1]
```diff
@@ -1,21 +1,23 @@
-#!/bin/bash
+!/bin/bash
```
Comment (steveb) [inline scripts/recipes/install_guitarix.sh]: whoops, will fix

**Comment by guysoft** [comment]
cc @jofemodo Suggest you look at this before it will become non-mergable. Has lots of changes that might be really good. Also nightly is working so we can test.

**Comment by jofemodo** [comment]
Hi @steveb !

I've been reviewing and it seems OK! I suppose you tested with Jessie ...
For working with Stretch, perhaps you should add the "xsltproc" to the "setup_system_rbpi_raspbian_lite_stretch.sh" too?

I will merge your changes and add the package ;-)

**Comment by steveb** [comment]
awesome, thanks!

**Pull request merged**